pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -319,15 +319,8 @@ let
     NIX_CFLAGS_COMPILE = "-Wno-unknown-warning-option";
 
     buildPhase = let
-      # Build paralelism: on Hydra the build was frequently running into memory
-      # exhaustion, and even other users might be running into similar issues.
-      # -j is halved to avoid memory problems, and -l is slightly increased
-      # so that the build gets slight preference before others
-      # (it will often be on "critical path" and at risk of timing out)
       buildCommand = target: ''
-        ninja -C "${buildPath}"  \
-          -j$(( ($NIX_BUILD_CORES+1) / 2 )) -l$(( $NIX_BUILD_CORES+1 )) \
-          "${target}"
```

Comment:
I think we should still respect `$NIX_BUILD_CORES`.

Reply:
Isn't it the default behavior?
Need to check.
If not, then `ninja` -> `ninja -j$NIX_BUILD_CORES` should be the treewide change

Reply:
It's the default if you don't call it explicitly; probably not this way.  It's possible that we can remove the command and just override the targets.

Reply:
To be clear, normally packages should not call ninja explicitly.  Then the hooks just do their job: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/ninja/setup-hook.sh

To make this cleaner, I expect it would be better not to override `buildPhase` and use `ninjaFlags(Array)` instead.

Reply:
I'd keep it explicit, without introducing hook magic, because `chromium` is a complex project and what can be build from its source tree is not limited to the browser executable.
There is also selenium driver, libwebrtc, ...
Magic let us write succinct code, but it is difficult to understand what is going under the hood and how to modify it (for example to build selenium driver along with the browser, or libwebrtc for Telegram Desktop)

Reply:
So I guess this is resolved now?